parsec/backend/cli/run.py

```diff
@@ -273,6 +273,15 @@ def handle_parse_result(
         " --backend-addr=parsec://localhost:<port>(?no_ssl=False if ssl is not set)`"
     ),
 )
+@click.option(
+    "--sse-keepalive",
+    default=30,
+    show_default=True,
+    type=int,
```

Comment:
I'd use `float` if the unit is seconds

Reply:
I wonder about int vs float here, but sub-second precision for this is overkill and using int saves us from float strangeness in parsing (passing `NaN`, `inf`, `-inf` etc.)

Reply:
I would not worry about that, `float` in seconds in the de-facto standard for time handling in python. Typically, `sse_keepalive` is used in `trio.move_on_after` which takes a `float`:

```python
trio.move_on_after(backend.config.sse_keepalive)
```

Reply:
I trust you on this, but my guts tell me float is just too clever for our own good :/
Anyway if float do get annoying, it would be easy (and won't break much of the compatibility) to switch to `int` here so no big deal ^^

Reply:
On the other hand, the good part of using float here is can use `math.inf` instead of `None` to disable the keepalive, hence not needing a special case for this \o/